### ISSUE TITLE
Target Layer for HiResCAM

### ISSUE THREAD

**Issue by manuelGue** [jacobgil/pytorch-grad-cam#379]
I understand the principles of CAM and the differences between GradCAM and HiResCAM, too. If I take the target layer for HiResCAM with layer4, I get the same results as with GradCAM. Is this related to the BatchNormalization layer behind the last Convolutional layer in the last block? Which target layer do I have to use for HiResCAM to get the expected results?

**Comment by jacobgil** [comment]
I think if you go one layer back, model.layer3[-1], it should start getting significantly different.
